downstream/modules/hub/proc-sync-image.adoc

```diff
@@ -13,24 +13,30 @@ You must have permission to view and pull from a private container repository.
 .Procedure
 
 To sync an image from a remote container registry, you need to configure a 
-remote registry.
+remote registry:
 
 . Navigate to menu:Execution Environments[Remote Registries].
 
 . Add https://registry.redhat.io to the registry.
 
 . Add any required credentials to authenticate. 
-
-NOTE: Some container registries are aggressive with rate limiting. It is  
-advisable to set a rate limit under Advanced Options.
-
++
+[NOTE]
+====
+Some container registries are aggressive with rate limiting. 
+It is advisable to set a rate limit under Advanced Options.
+====
++
 . Navigate to menu:Execution Environments[Execution Environments].
```

Comment:
Suggest rewording to the following for consistency across the library:
From the navigation panel, select ...